pkg/api/plugins.go

```diff
@@ -29,6 +29,7 @@ func (hs *HTTPServer) GetPluginList(c *models.ReqContext) response.Response {
 	enabledFilter := c.Query("enabled")
 	embeddedFilter := c.Query("embedded")
 	coreFilter := c.Query("core")
+	catalogFilter := c.Query("catalog")
```

Comment:
**Question:** is there any reason to have this filter?

I am only asking because I guess once we have this config option then the user probably only wants a single option to set in order hide a plugin.

Reply:
This is my main concern, I haven't gone through what in the frontend using this endpoint that is not the catalog and still need to process hidden plugins. Do you have any ideas?

Reply:
With a specific endpoint in the backend for the catalog, serving both installed and plugins from gcom, we don't need this filter.

Reply:
Short term solution, lets do the filtering in the frontend so that we can filter out both installed and gcom plugins